src/app/home/home.component.html

```diff
@@ -26,7 +26,7 @@ <h1 text-center>{{ programName }}</h1>
 
     <ng-container *ngIf="todoItems.length; else noItems">
       <div *ngFor="let todoItem of todoItems">
-        <app-todo-card [todoItem]="todoItem"></app-todo-card>
+        <app-todo-card [todoItem]="todoItem" (click)="goToReview(todoItem.contextId,todoItem.id)"></app-todo-card>
```

Comment:
Check what is the "id" in the review routing, it's not the todo item id

Reply:
You put `todoItem.contextId` here, but there's no `contextId` in the home service. 
Also, since the `context_id` will be in the meta of todo item, it should be `todoItem.meta.contextId`

Reply:
But since these are related to home.service, we will change them when we implement home service. Will just leave it like this for now.